doc/man7/migration_guide.pod

```diff
@@ -60,6 +60,12 @@ There is however a dedicated C<install_fips> make target, which serves the
 special purpose of installing only the FIPS provider into an existing
 OpenSSL installation.
 
+Not all algorithms may be available for the application at a particular moment.
```

Comment:
What about this alternative text:

> Due to the change to providers for implementations, it cannot be assumed that algorithms are always available.  For example, an application that uses a cipher or digest algorithm via the EVP interface will need to verify the result of the corresponding L<EVP_EncryptInit(3)>,
L<EVP_EncryptInit_ex(3)> or L<EVP_DigestInit(3)> function call.  These functions return a failure when the requested algorithm is not available.

Reply:
Well... the init() calls might have failed even on 1.1.1 in certain rare circumstances. So it was always a very good idea to check the return value.

Reply:
So I slightly prefer keeping the text as is.